tools/lint/src/main/kotlin/ThreadPoolDetector.kt

```diff
@@ -19,7 +19,15 @@ class ThreadPoolDetector : Detector(), SourceCodeScanner {
         "newScheduledThreadPool",
         "newSingleThreadExecutor",
         "newSingleThreadScheduledExecutor",
-        "newWorkStealingPool"
+        "newWorkStealingPool",
+        "<init>"
```

Comment:
Is having `<init>` as a method target here actually linting `<init>` invocations? It looks like you're handling constructors with a different method.